astro.config.mjs

```diff
@@ -8,6 +8,7 @@ export default defineConfig({
 	site: 'https://blinklabs.io',
 	integrations: [
 		starlight({
+			favicon: './src/assets/icononly_transparent_nobuffer.png',
```

Comment:
I would grab this one, to ensure consistency: https://github.com/blinklabs-io/www/blob/main/public/favicon.ico